app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
-<span class="badge"><%= table.scenario.tier %></span>
+<!-- Trigger the modal with a button -->
+<button type="button" class="btn btn-round-xs btn-tier btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_tier"><%= table.scenario.tier %></button>
+<!-- Modal -->
+<div id="table_<%= table.id %>_tier" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">This is a tier <%= table.scenario.tier %> scenario.</h4>
+      </div>
+      <div class="modal-body">
+        <p>You must have a character that is within the levels of <%= table.scenario.tier %> to play.<br>
+          If this is an interactive special, you can play a character on subtier higher or lower, as they are ticketed
+          by sub-tier instead of full tier, though we recommend strongly remaining within your sub-tier.
+        </p>
```

Comment:
Reword: This scenario is for characters within the levels of <%= table.scenario.tier %>.<br> If this is an interactive special, characters <em>can</em> play on a subtier that is one higher or lower; however, we strongly recommend signing up within the sub-tier.